docs/source/examples/vision/checks/distribution/source/plot_train_test_label_drift.py

```diff
@@ -18,48 +19,41 @@
 
 Causes of label drift include:
 
-* Natural drift in the data, such as lighting (brightness) changes between summer and winter.
+* Natural drift in the data, such as a certain class becoming more prevalent in the test set.
+  For example, cronuts becoming more popular in a food classification dataset.
 * Labeling issues, such as an analyst drawing incorrect bounding boxes for an object
   detection task.
 
-It may be the case that the change in the distribution of the label is due solely
```

Comment:
Agree that it's too long, but given the prevalence of drift in our package, do you think we need a "drift user guide" somewhere, delving deeper into they why when and how of drift?

Reply:
I agree. Here's a task - https://github.com/deepchecks/deepchecks/issues/1066